_includes/tutorials/ksql-nested-json/ksql/code/Makefile

```
@@ -0,0 +1,15 @@
+STEPS_DIR := tutorial-steps
+DEV_OUTPUTS_DIR := $(STEPS_DIR)/dev/outputs
+TEST_OUTPUTS_DIR := $(STEPS_DIR)/test/outputs
+PROD_OUTPUTS_DIR := $(STEPS_DIR)/prod/outputs
+TEMP_DIR := $(shell mktemp -d)
+SEQUENCE := "dev, test, prod, ccloud"
+
+tutorial:
+	rm -r $(DEV_OUTPUTS_DIR) || true
+	rm -r $(TEST_OUTPUTS_DIR) || true
+	mkdir $(DEV_OUTPUTS_DIR)
+	mkdir -p $(TEST_OUTPUTS_DIR)
+	harness-runner ../../../../../_data/harnesses/ksql-nested-json/ksql.yml $(TEMP_DIR) $(SEQUENCE)
+	diff --strip-trailing-cr $(STEPS_DIR)/dev/expected-transient-reporting.log $(DEV_OUTPUTS_DIR)/transient-reporting/output-0.log
```

```suggestion
	diff --strip-trailing-cr $(STEPS_DIR)/dev/expected-transient-reporting.log $(DEV_OUTPUTS_DIR)/transient-reporting/output-0.log
	diff --strip-trailing-cr $(STEPS_DIR)/test/expected-results.log $(TEST_OUTPUTS_DIR)/test-results.log
```
Can we add in the suggestion above? That way we verify that the test statement section is working as expected in our pipeline.

---

ack updated